config/deploy.rb

```diff
@@ -0,0 +1,9 @@
+set :application, 'apprenticeship.subvisual.co'
+set :repo_url, 'git@github.com:groupbuddies/apprenticeship.git'
+set :stage, :production
+set :branch, :master
+
```

Comment:
please add `set :linked_dirs, %w(log)`

i've configured nginx to log everything inside /apprenticeship to that directory